responses/de/HassGetState.yaml

```diff
@@ -61,3 +61,25 @@ responses:
 
       wie_viele: |
         {{ query.matched | length }}
+
+      wo: |
+        {% if state.state == "not_home" %}
+          {{ slots.name | capitalize }} ist nicht da
+        {% else %}
+          {{ slots.name | capitalize }} ist im Bereich {{ state.state }}
```

Comment:
Proposal: if a person is in an area for quite a long time (to be defined...), provide the duration in the response. I've been using a custom_sentence for "where is xyz" for quite some time now and me and my family really appreciate when the response is something like "Maria ist seit 6 Stunden im Bereich Büro" or "Frank ist seit 1 Stunde im Supermarkt". Adds some unforeseen value ("oh, then he must be back soon").

Reply:
(_at the moment_) we only have the current state available, not the "last updated" or "laste changed" attribute.
Further adding this needs a discussion on the language-leader board.